README.md

```diff
@@ -7,3 +7,5 @@
 
 
 Edit the header at `header.js` and footer at `footer.js` at the root.
+
+Update About Us page by running `about\script.py`.
```

Comment:
Shouldn't this be `about/script.py`? :sweat_smile:

Reply:
Not unless you're using windows PC 😅. My bad. Should I change it?

Reply:
Definitely. It's outright "incorrect".
 @mukul-mehta seems like you guys aren't grooming your juniors well :wink:

Reply:
@americast Apologies. I've changed it.

Reply:
Sorry, I didn't have a look at this. Just tested the code. Will be sure to do that from next time :smile:

Reply:
@mukul-mehta code review is something else. I would want to stress on proper grooming 😛:wink: